dataretrieval/nwis.py

```diff
@@ -455,17 +455,23 @@ def get_info(**kwargs):
         siteOutput=expanded cannot be used if seriesCatalogOutput=true or with
         any values for outputDataTypeCd.
 
-    seriesCatalogOutput : boolean
+    seriesCatalogOutput : boolean or string
```

Comment:
Better to stick with boolean and convert to string behind the scenes, don't you think?

Reply:
Yes, I agree with this, it is more clear if we document this as expecting a boolean `True` value.